README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/neuropoly/axondeepseg/master?filepath=notebooks%2Fgetting_started.ipynb)
 [![Build Status](https://api.travis-ci.org/neuropoly/axondeepseg.svg?branch=master)](https://travis-ci.org/neuropoly/axondeepseg)
+[![Documentation Status](https://readthedocs.org/projects/axondeepseg/badge/?version=v0.4)](http://axondeepseg.readthedocs.io/en/v0.4/?badge=v0.4)
```

Comment:
do we need to specify a version? can't we use something like "master"? otherwise we need to remember to update that number if we change the version, no?